source/extensions/filters/http/jwt_authn/extractor.cc

```diff
@@ -186,6 +188,8 @@ class ExtractorImpl : public Logger::Loggable<Logger::Id::jwt>, public Extractor
   absl::btree_map<std::string, CookieLocationSpec> cookie_locations_;
 
   std::vector<LowerCaseString> forward_payload_headers_;
+
+  std::vector<LowerCaseString> claim_to_headers_;
```

Comment:
combined to use single vector, call it `headers_to_sanitize`,  add both `forward_payload_headers` and `claim_to_headers` to it.

also rename the function santizePayloadHeaders to sanitizeHeaders

Reply:
Sure